packages/manager/modules/exchange/src/domain/mx-autoconfig/domain-mx-autoconfig.controller.js

```diff
@@ -25,26 +25,47 @@ export default class ExchangeDomainMxAutoconfigCtrl {
 
     this.$routerParams = wucExchange.getParams();
     this.domain = navigation.currentActionData;
+    this.domainDiag = {};
```

Comment:
```suggestion
    this.domainDialog = {};
```

Reply:
domainDiag was not my choice, was already defined like that before.
For info, it's not mean Dialog but Diagnose ;)